great-idea/css/index.css

```
@@ -63,4 +63,234 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+
+.navigation {
+
+	display: flex;
+	flex-wrap: nowrap;
+	flex-direction: row;
+	justify-content: center;
+	
+}
+
+a {
+	margin: 30px;
+	font-weight: bold;
+	font-size: 15px;
+	padding-top: 24px;
```

When you're using padding on more than one side, use padding: (top) (right) (bottom) (left). The same pattern applies to margins.
```suggestion
	padding: 24px 32px 16px 0;
```